src/js/components/Video/Video.js

```diff
@@ -254,10 +274,71 @@ const Video = forwardRef(
         onClick: () => showCaptions(caption.active ? -1 : 0),
       }));
 
+      const buttonTable = {
```

Comment:
For all of these, I think we can include the icon property here as well, instead of calculating it and then adding it later on.